packages/vscode/CONTRIBUTING.md

```diff
@@ -72,6 +72,50 @@ When running the End-to-End tests in GitHub Actions before publishing, the scrip
 The End-to-End tests that are run after the publish of the extension are located in `scripts/e2eTestsOnVsix/test.sh`.
 In both cases the tests in `packages/vscode/src/__test__` with the schmeas located in `packages/vscode/fixtures` are used.
 
+## Pull Requests
+
+When a PR is opened, the "PR Build extension" GitHub Action will build and upload a `pr<PR_NUMBER>-prisma.vsix` file and link to it in a comment (which will be updated for each commit).
+
+### How to install and use this PR Build version:
+
+#### With the UI
+
+- In the extensions tab, filter the Prisma extenions with `@installed prisma`
+- Disable all the Prisma extensions (Prisma & Prisma Insider)
+- From the ... menu or the command palette, click "Install from VSIX..."
+
+#### With the command line
+
+```bash
+code --disable-extension Prisma.prisma && code --disable-extension Prisma.prisma-insider`
+
+# Replace with the correct PR number
+wget "https://github.com/prisma/language-tools/blob/artifacts/pull-request-artifacts/pr<PR_NUMBER>-prisma.vsix?raw=true"
+
+code --install-extension pr<PR_NUMBER>-prisma.vsix
+```
+
+Now the extension can be tested: open a `schema.prisma` file.
+
+### After testing you might want to clean up things a bit
+
+#### With the UI
+
+- In the extensions tab, filter the Prisma extenions with `@installed prisma`
+- Right click the `Prisma Insider - PR <PR_NUMBER> build` and click `Uninstall`
+- Enable the Prisma or Prisma Insider extension
+
+#### With the command line
+
+```bash
+rm pr<PR_NUMBER>-prisma.vsix
+code --uninstall-extension Prisma.prisma-insider-pr-build
+# For the latest public version
+code --install-extension Prisma.prisma
+# Or for Insider
+code --install-extension Prisma.prisma-insider
+```
```

Comment:
I'd like to test this feature on Sublime Text as well, but I can't find `bin.js` (https://github.com/prisma/language-tools/blob/main/packages/language-server/package.json#L17) in the attached `vsix` file ? (which I assume is the entry point for the LS).

Reply:
@Ultra-Instinct-05 Sure! You will be able to try it once it's merged. Because right now it's using the local language-server, once merged it will be published at https://www.npmjs.com/package/@prisma/language-server

Reply:
@Jolg42 I actually meant helping test it before it's released 🙂 Anyways I pulled & built the changes of this PR onto my local fork and wired up this PR's LS changes to ST and the composite type completions seems to be working great !

One small issue is when I manually delete the completion and try to trigger it by typing, it doesn't seem to work ? As an example, as you see in the GIF, `alpha` is auto completed after typing `.`, but when I manually delete `alpha` and try to type `alpha` again, it's not suggested.

![composite_bug](https://user-images.githubusercontent.com/34304644/166729452-4a29a1d2-21fc-4eea-85cf-31515bddef77.gif)

Reply:
Oh 😊 thanks for testing it! 💚
Yes it's the same in VS Code, I think we can take a look at this separately to improve the suggestion behavior.
I need to merge this PR to continue some other work first 😄